app/client/src/components/pages/State.Tabs.AdvancedSearch.js

```diff
@@ -1147,62 +1151,63 @@ function AdvancedSearch() {
 }
 
```

Comment:
This could be extracted out, too, similar to the `VirtualizedList` file
```suggestion
function MenuItem({
  index,
  width,
  setSize,
  value,
}: {
  index: number,
  width: number,
  setSize: (index: number, size: number) => void,
}) {
  const rowValue = value;

  const rowRef = useRef();

  // keep track of the height of the rows to autosize rows
  useEffect(() => {
    if (!rowRef?.current) return;

    setSize(index, rowRef.current.getBoundingClientRect().height);
  }, [setSize, index, width]);

  return <div ref={rowRef}>{rowValue}</div>;
}

```

Reply:
Good catch. I went ahead and moved that function out into a separate component.